Whiskey/Functions/Resolve-WhiskeyPowerShellModule.ps1

```diff
@@ -36,6 +36,10 @@ function Resolve-WhiskeyPowerShellModule
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
+    Import-WhiskeyPowerShellModule -Name 'PackageManagement', 'PowerShellGet'

```

Comment:
Yikes. That's important, otherwise the system-installed modules are auto-loaded! Good catch.